lib/modules/platform/gitea/index.ts

```diff
@@ -495,6 +496,33 @@ const platform: Platform = {
         labels: labels.filter(is.number),
       });
 
+      if (platformOptions?.usePlatformAutomerge) {
+        if (semver.gte(defaults.version, '1.17.0')) {
+          try {
+            await helper.mergePR(config.repository, gpr.number, {
+              // TODO: pass strategy (#16884)
+              Do: config.mergeMethod,
+              merge_when_checks_succeed: true,
+            });
+
+            logger.debug(
+              { prNumber: gpr.number },
+              'Gitea-native automerge: success'
+            );
+          } catch (err) {
+            logger.warn(
+              { err, prNumber: gpr.number },
+              'Gitea-native automerge: fail'
+            );
+          }
+        } else {
+          logger.debug(
+            { prNumber: gpr.number },
+            'Gitea-native automerge: not supported on this Gitea version. Requires >=1.17.0'
```

Comment:
```suggestion
            'Gitea-native automerge: not supported on this version of Gitea. Use 1.17.0 or newer.'
```

How about this? It think it explains better what the user can do to get things working: 

> Use version 1.17.0 or newer.

Reply:
I've opened a PR to change the debug message for GitHub:

- #16889